buildpacks/use-volume/bin/build

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+echo "---> Hello World buildpack"
+
+# INPUT ARGUMENTS
+env_dir=$2/env
+layers_dir=$1
+plan_path=$3
+
+# ENV VARS
+echo "     env_dir: ${env_dir}"
+echo "     env vars:"
+if compgen -G "${env_dir}/*" > /dev/null; then
+  for var in ${env_dir}/*; do
+    declare "$(basename ${var})=$(<${var})"
+  done
+fi
+export | sed 's/^/       /'
+
+# VOLUMES
+echo "     reading volume contents:"
+echo "         $(cat /cnb/volume/volume_contents.txt)"
```

Comment:
I don't think we should be encouraging mounting to an unsafe area. A safe area that we should encourage: `/platform`

Additionally, if this is the only difference from the hello world buildpack, I would suggest we simply add this to it instead of introducing another buildpack.